ci-scripts/check_pr_modified_files_for_oai_pipeline.py

```diff
@@ -20,6 +20,9 @@
 mme_is_impacted = False
 
 for line in list_files.split('\n'):
+    res = re.search('lte/gateway/c/core/BUILD.bazel', line)
```

Comment:
This won't catch all changes to MME and SCTPD is defined in another file.

You could look at the [bazel.yml](https://github.com/magma/magma/blob/eb5fdbe8d038b136d422e5e49f5154817e50ac34/.github/workflows/bazel.yml) workflow for inspiration. 

- Option 1: Rough path filter, e.g. on
```
  - '**/BUILD'
  - '**/*.BUILD'
  - '**/*.bazel'
  - '**/*.bzl'
  - '.bazelrc'
```
- Option 2: Using Bazel-diff, see the `bazel_diff` job.
  - Here you would need to run `bazel/scripts/bazel_diff.sh <newer_commit> <older_mergebase_commit>` and check if the output contains the MME/SCTPD targets.
  - Additionally you need to check if the `.bazelrc` or `WORKSPACE.bazel` files have been touched, as they are on a meta-level.

Reply:
The option 2 could replace the job trigger in general, as you would know exactly when there is a change to MME/SCTPD. 

Option 1 would only be an addition to the current triggers.

Reply:
This is perhaps a minimal approach that should catch most cases - in particular the issue that just happened:

```
/WORKSPACE.bazel
/.bazelrc
/lte/gateway/c/sctpd/**/BUILD.bazel
/lte/gateway/c/core/**/BUILD.bazel
/bazel/
```